docs/running/server/wes.rst

```diff
@@ -109,14 +129,28 @@ To run the server on a Toil leader instance on EC2:
 
 #. Launch a Toil cluster with the ``toil launch-cluster`` command with the AWS provisioner
 
-#. SSH into your cluster with the ``--sshOption=-L8080:localhost:8080`` option to forward port ``8080``
+#. SSH into your cluster with the ``--sshOption=-L8080:localhost:8080`` option to forward port ``8080`` to
+   ``localhost:8080``
```

Comment:
This is a little confusing; that `localhost` is evaluated relative to the remote machine, but that's not quite clear from the new description.